app/assets/stylesheets/application.scss

```diff
@@ -258,30 +257,86 @@ header {
 }
 
 /* Media Query for Mobile */
-@media screen and (max-width: 768px) {
-  .debt-show article {
-    grid-template-columns: 1fr;
-    text-align: center;
-  }
+@media only screen and (min-width: 1024px) {
+  .home-parent-div {
+    display: flex;
+    height: 100vh;
 
-  .debt-show h5 {
-    font-size: 1.5rem;
-  }
+    header {
+      background: #259672;
+      width: 40%;
+      border-radius: 0;
+      box-shadow: 6px 4px 6px rgba(0.5, 0.5, 0.5, 0.1);
+      height: inherit;
 
-  .debt-show p {
-    font-size: 1.25rem;
-  }
+      h1 {
+        font-size: 3rem;
+      }
+    
+      .para {
+        font-size: 1.325rem;
+      }
+    }
 
-  .debt-show .edit-link,
-  .debt-show form {
-    display: block;
-    text-align: center;
-    margin-top: 1rem;
-  }
+    .main-footer {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      width: 60%;
 
-  .debt-show i {
-    font-size: 1.5rem;
+      .index-footer {
+        position: static;
+      }
+    
+      .footer {
+        width: 60%;
+    
+        button {
+          background-color: #259672;
+          color: #fff;
+          border: none;
+          width: 80%;
+          padding: 12px;
+          margin-bottom: 10px;
+          border-radius: 20px;
+          font-size: 1.2rem;
+          cursor: pointer;
+        }
+
+        button:hover {
+          background-color: #7ce2c0;
+          transition: all 1s ease-in;
+          color: #064d36;
+        }
+      }
+    }
   }
+  
+
+  // .debt-show article {
```

Comment:
Be bold! Delete instead of commenting out. If you're really worried, do it in two commits, add then delete, and it will live in the git history forever.

Reply:
Thank you @andynu for the feedback, I will delete this.